dot/parachain/provisioner/prioritized_disputes_selection.go

```diff
@@ -19,6 +18,70 @@ type BlockState interface {
 	GetRuntime(blockHash common.Hash) (instance parachain.RuntimeInstance, err error)
 }
 
+// IsVoteWorthToKeep determines if a vote is worth to be kept, based on the onchain disputes.
+func IsVoteWorthToKeep(
+	validatorIndex parachaintypes.ValidatorIndex,
+	disputeStatement parachaintypes.DisputeStatement,
+	onchainState parachaintypes.DisputeState,
+) bool {
+	var offchainVote bool
+	var validKind *parachaintypes.ValidDisputeStatementKind
+
+	statement, err := disputeStatement.Value()
+	if err != nil {
+		panic("unexpected empty inner in DisputeStatement")
```

Comment:
will panic terminate the node? or that is the desired behavior

Reply:
yeah, these values represent rust enums, and basically when decoding from raw bytes to a golang type they should not be empty, they should always have a variant to return from method `Value`.